autospec/tarball.py

```diff
@@ -499,10 +471,62 @@ def process_go_archives():
         if url_mod not in archives:
             go_archives.extend([url_mod, ':'])
         if url_zip not in archives:
-            go_archives.extend([url_zip, f'{ver}/'])
+            go_archives.extend([url_zip, ''])
```

Comment:
Could you explain why you are dropping ver here?

Reply:
Yes, basically because autospec don't use it.

Currently, it just add another dir:  v0.3.1/github.com/BurntSushi/toml@v0.3.1
With this patch: github.com/BurntSushi/toml@v0.3.1

It's important to notice that this is just for source exploration (finding dependencies, license, etc). At the end, it has no impact on the specfile so I have no problem to add it back.

Reply:
I think I was probably just being overly paranoid about conflicts though maybe it was due to a refactor making it no longer needed. I'm fine with dropping this considering the url uniqueness is reasonably certain.